### PR TITLE
doco improvements

### PR DESCRIPTION
**Pull request by Nathanator**
Fixing monospacing
Cleaned up some whitespace
Added base section on running code

### CODE REVIEW

File: docs/devguide/flashfirmware.rst [hunk 1]
```diff
@@ -33,15 +32,17 @@ some convenience targets.
...
 ---------------------------------------
 
-tools/makecombined
-
-hexlify
-
+Try these::
 
+  tools/makecombined
+  hexlify
```
Comment (dpgeorge) [inline docs/devguide/flashfirmware.rst]: I think with makecombined.py you don't need hexlify.py, so the latter doesn't need to be mentioned.  You could actually mention in more detail how to use makecombined, eg: ./tools/makecombined.py build/bbc-microbit-classic-gcc-nosd/source/microbit-micropython.hex myscript.py

Comment (Nathanator) [inline docs/devguide/flashfirmware.rst]: Cheers, didn't realise that existed.  I'll update.

**Comment by dpgeorge** [comment]
@Nathanator thanks for fixing, but can you please rebase to the latest master so that the commit is clean?  Thanks!

**Comment by Nathanator** [comment]
Will do.  Git amateur, unfortunately, but I'm sure I can work it out :-)

**Pull request closed without merging**